dist/formats/finder/frontend/schema.json

```diff
@@ -597,6 +597,12 @@
             "type": "string"
           }
         },
+        "links": {
```

Comment:
I think this might need to be singular `link` (like the content purpose supergroup one above) as it gets transformed to `reject_link` in the query

Reply:
Ah no problem, should I also change the <policies> above to <policy>?

Reply:
Done, it all says link instead now 👍